src/cookieutils.ts

```diff
@@ -67,7 +43,10 @@ export class Cookie {
     }
 }
 
-function writeCookie(details: CookieDetails) {
-    const {name, value, expires, domain} = details;
-    document.cookie = `${name}=${value}${expires}; path=/; domain=${domain}; SameSite=Lax`;
+function writeCookie(name: string, value: string, expirationDate?: Date, domain?: string) {
+    document.cookie =
+        `${name}=${value}` +
+        (expirationDate ? `;expires=${(expirationDate as any).toGMTString()}` : '') +
```

Comment:
You can use `toUTCString`, which will remove the need for the `any` cast.

Explanation: 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toUTCString#aliasing

```suggestion
        (expirationDate ? `;expires=${expirationDate.toUTCString()}` : '') +
```